src/org/labkey/test/BaseWebDriverTest.java

```diff
@@ -1552,6 +1558,30 @@ public String getModulePropertyValue(ModulePropertyValue property)
         return (String)ref.getValue();
     }
 
+    public void setResponseServerConfigurations(LinkedHashMap<String, String> props)
```

Comment:
I'll let @labkey-tchad weigh in on this, but it doesn't seem right that `BaseWebDriverTest` should know anything about the response server config details. It knows about module properties, but that's a very generic mechanism.